Sources/FluentKit/Properties/Timestamp.swift

```diff
@@ -244,6 +249,14 @@ extension TimestampFormat {
     public static let iso8601 = TimestampFormat("iso8601", formatter: ISO8601DateFormatter.init)
 }
 
+extension TimestampFormat {
+    public static let iso8601WithMilliseconds = TimestampFormat("iso8601WithMilliseconds", formatter: {
+        let formatter = ISO8601DateFormatter.init()
```

Comment:
Why use `.init()` here.
```suggestion
        let formatter = ISO8601DateFormatter()
```

Reply:
Because my copy-pasta was lazy, no other reason 🙂